### PR TITLE
docs: pipeline overview and how to configure pipes

### PR DESCRIPTION
**Pull request by rhiaro**
And more structure to pipeline docs. Move content
from 'stage' docs where applicable.

### CODE REVIEW

**Comment by rhiaro** [comment]
[Document how to configure normalisation pipeline / switch bits on or off](https://trello.com/c/RNd8EPkT/96-document-how-to-configure-normalisation-pipeline-switch-bits-on-or-off)

File: docs/stage/normalise-data.md [hunk 1]
```diff
@@ -1,15 +1,8 @@
 # Normalise Data
 
-The tool will normalise raw data by passing it through a series of pipes.
+The tool will normalise raw data by passing it through a series of pipes. See [Pipeline](pipeline) for more information.
```
Comment (odscjames) [inline docs/stage/normalise-data.md:3]: Is that path correct?

File: docs/stage/normalise-data.md [hunk 2]
```diff
@@ -1,15 +1,8 @@
...
 
-Pipes are called in the order defined in `src/lib/pipes/index.js` and are called once for each bit of raw data.
-
-Before the first pipe is called, there is an array of normalised data which is empty.
-Each pipe has access to the original data and the normalised data created so far.
-Each pipe can delete, edit or create normalised data as it wants.
-After all pipe are called, whatever normalised data is left is saved to the database.
-
-The normalised data is stored in the `normalised_data` table.
+After all pipe are called, the resulting normalised data is left is saved to the database, in the `normalised_data` table.
```
Comment (odscjames) [inline docs/stage/normalise-data.md:5]: After all pipe*s* :-)

Comment (odscjames) [reply docs/stage/normalise-data.md:5]: "data left"

**Comment by rhiaro** [comment]
[Visualise / document [data structures, data merging, error handling, system] . architecture](https://trello.com/c/ZcD1tYsR/38-visualise-document-data-structures-data-merging-error-handling-system-architecture)

**Pull request merged**